### PR TITLE
ISPN-6838 Added info about versioning

### PR DESCRIPTION
**Pull request by slaskawi**
https://issues.jboss.org/browse/ISPN-6838

### CODE REVIEW

**Comment by infinispanrelease** [comment]
Started - TeamCity Infinispan :: Docker image Pull requests JDK8 [Build 19](http://ci.infinispan.org/viewLog.html?buildId=41120&buildTypeId=Infinispan_DockerImagePullRequestsJdk8&guest=1) for ae89acdfb50dd727b3e3750957e3cad12b81202b

**Comment by slaskawi** [comment]
//cc @goldmann @gustavonalle - After reading the [automated build guide](https://docs.docker.com/docker-hub/builds/), it looks like we need to store each version in its own branch. 

Alternatively we could change the configuration in Docker hub and create one generic branch (called `8.2.x` and push all minors there). 

WDYT?

**Comment by goldmann** [comment]
Your readme is wrong - you talk about `8.2.0` there.

**Comment by goldmann** [comment]
Change it to majo only `8.2.x` and maybe add a few words about tags.

**Comment by gustavonalle** [comment]
@slaskawi I wrote my proposal in the JIRA

**Comment by slaskawi** [comment]
Okey dokey, understood, let me update it.

**Comment by slaskawi** [comment]
@gustavonalle @goldmann Updated. Please have a look

**Comment by infinispanrelease** [comment]
Started - TeamCity Infinispan :: Docker image Pull requests JDK8 [Build 20](http://ci.infinispan.org/viewLog.html?buildId=41121&buildTypeId=Infinispan_DockerImagePullRequestsJdk8&guest=1) for cb53524b560155cc58f298f57c605db6f0ef1992

File: README.md [hunk 1]
```diff
@@ -10,7 +10,11 @@ This repository contains Docker imgaes for different flavours of Infinispan:
 
 Infinispan can be used in two different ways: in client-server mode and in embedded mode.  In client-server mode Infinispan runs as standalone server in diferent JVM and application using Infinispan connects to the server as a client. On the other hand, in embedded mode Infinispan runs in the same JVM as application and application access it as a library. For more details see [Infinispan documentation](http://infinispan.org/docs/7.0.x/user_guide/user_guide.html#_server_modules).
 
+## Branching strategy
 
+[Master branch](https://github.com/jboss-dockerfiles/infinispan/tree/master) contains current development version. Each major stable version has its own branch (e.g. [8.2.x](https://github.com/jboss-dockerfiles/infinispan/tree/8.2.x)). All minor versions are tagged from that branch (e.g. 8.2.1.Final, 8.2.2.final).
```
Comment (goldmann) [inline README.md]: As long as "8.2.1.Final" is for you minor. I would call it micro :) Same goes to "major" above.

Comment (slaskawi) [inline README.md]: yeah, you are right... update on the way :)

Comment (slaskawi) [inline README.md]: Updated

**Comment by infinispanrelease** [comment]
Started - TeamCity Infinispan :: Docker image Pull requests JDK8 [Build 21](http://ci.infinispan.org/viewLog.html?buildId=41122&buildTypeId=Infinispan_DockerImagePullRequestsJdk8&guest=1) for 34f401cdfa635d6eb1584e6efe11a03d64715060

**Comment by goldmann** [comment]
LGTM

**Comment by gustavonalle** [comment]
:+1:

**Comment by infinispanrelease** [comment]
**UNKNOWN** - TeamCity Infinispan :: Docker image Pull requests JDK8 [Build 21](http://ci.infinispan.org/viewLog.html?buildId=41122&buildTypeId=Infinispan_DockerImagePullRequestsJdk8&guest=1) for 34f401cdfa635d6eb1584e6efe11a03d64715060
Tests: 0, 0 failed (0 new), 0 ignored. Build time: 00:18:00

Error message is logged

**Comment by infinispanrelease** [comment]
**UNKNOWN** - TeamCity Infinispan :: Docker image Pull requests JDK8 [Build 19](http://ci.infinispan.org/viewLog.html?buildId=41120&buildTypeId=Infinispan_DockerImagePullRequestsJdk8&guest=1) for ae89acdfb50dd727b3e3750957e3cad12b81202b
Tests: 0, 0 failed (0 new), 0 ignored. Build time: 02:56:41

**Comment by infinispanrelease** [comment]
**UNKNOWN** - TeamCity Infinispan :: Docker image Pull requests JDK8 [Build 20](http://ci.infinispan.org/viewLog.html?buildId=41121&buildTypeId=Infinispan_DockerImagePullRequestsJdk8&guest=1) for cb53524b560155cc58f298f57c605db6f0ef1992
Tests: 0, 0 failed (0 new), 0 ignored. Build time: 02:32:57

**Comment by infinispanrelease** [comment]
Started - TeamCity Infinispan :: Docker image Pull requests JDK8 [Build 23](http://ci.infinispan.org/viewLog.html?buildId=41124&buildTypeId=Infinispan_DockerImagePullRequestsJdk8&guest=1) for 34f401cdfa635d6eb1584e6efe11a03d64715060

**Comment by infinispanrelease** [comment]
**SUCCESS** - TeamCity Infinispan :: Docker image Pull requests JDK8 [Build 23/slaskawi:ISPN-6838/Info_about_versioning](http://ci.infinispan.org/viewLog.html?buildId=41124&buildTypeId=Infinispan_DockerImagePullRequestsJdk8&guest=1) for 34f401cdfa635d6eb1584e6efe11a03d64715060
Tests: 0, 0 failed (0 new), 0 ignored. Build time: 00:03:31

**Pull request merged**